libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/utils.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ErrorAnalysisOptions } from "@responsible-ai/error-analysis";
+import { localization } from "@responsible-ai/localization";
+
+import { GlobalTabKeys } from "../../ModelAssessmentEnums";
+import { FeatureImportancesTabOptions } from "../FeatureImportances";
+
+import { ITabsViewProps } from "./TabsViewProps";
+
+export function getInfo(
+  tabKey: GlobalTabKeys,
+  props?: ITabsViewProps,
+  errorAnalysisOption?: ErrorAnalysisOptions,
+  featureImportanceOption?: FeatureImportancesTabOptions
+): string[] {

```

Comment:
what about create an interface with `{body: string; title: string}`? So we can replace `getInfo(...)[0]` with `getInfo(...).body`, which is easier to read?

Reply:
updated, thanks!